automated-testing/readme.md

```diff
@@ -1,25 +1,56 @@
-| **Scenarios** | Developer perspectives | Developers and IT Pro tester perspectives | Developer and QA tester perspectives | End-user perspectives |
-| **When** | After each build | After Unit testing | Before E2E testing and after Unit and Integration testing | After System testing |
-**Automated or Manual** | Automated | Manual or automated  | Manual or automated | Manual |
+To use the table, either eyeball-browse or search for keywords.  The  
+
+| When I am working on... | I want to get this outcome... | ...so I should consider |
+|--|--|--|
+| Development | Prove backward compatibility with existing callers and clients | Shadow testing |
+| Development; [Integration testing](integration-testing/readme.md) | Ensure telemetry is sufficiently detailed and complete to trace and diagnose malfunction in [End-to-End testing](e2e-testing/readme.md) flows | Distributed Debug challenges; Orphaned call chain analysis |
+| Development | Ensure program logic is correct for a variety of expected, mainline, edge and unexpected inputs | [Unit testing](unit-testing/readme.md); Functional tests; [Integration testing](integration-testing/readme.md) |
+| Development | Prevent regressions in logical correctness; earlier is better | [Unit testing](unit-testing/readme.md); Functional tests; [Integration testing](integration-testing/readme.md); Rings (each of these are expanding scopes of coverage) |
+| Development | Quickly validate mainline correctness of a point of functionality (e.g. single API), manually | Smoke testing Tools: postman, powershell, curl |
+| Development; [Integration testing](integration-testing/readme.md) | Validate that multiple components function together across multiple interfaces in a call chain, incl network hops | [Integration testing](integration-testing/readme.md); End-to-end ([End-to-End testing](e2e-testing/readme.md)) tests; Segmented end-to-end ([End-to-End testing](e2e-testing/readme.md)) |
+| Development | Prove disaster recoverability – recover from corruption of data | DR drills |
+| Development | Find vulnerabilities in service Authentication or Authorization | Scenario (security) |
+| Development | Prove correct RBAC and claims interpretation of Authorization code | Scenario (security) |
+| Development | Document and/or enforce valid API usage | [Unit testing](unit-testing/readme.md); Functional tests|
+| Development | Prove implementation correctness in advance of a dependency or absent a dependency | [Unit testing](unit-testing/readme.md) (with mocks); [Unit testing](unit-testing/readme.md) (with emulators) |
+| Development | Ensure that the user interface is accessible | Accessibility |
+| Development | Ensure that users can operate the interface | [UI testing](ui-testing/readme.md) (human usability observation) |
+| Development | Prevent regression in user experience | UI automation; [End-to-End testing](e2e-testing/readme.md) |
+| Development | Detect and prevent &#39;noisy neighbor&#39; phenomena | Load |
```

Comment:
We do have a load testing section we can link to. It's structured a different as it's grouped under performance testing: performance-testing/load-testing.md

Reply:
fixed.